core/services/pipeline/task.eth_tx_test.go

```diff
@@ -23,6 +24,8 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/pipeline"
 )
 
+type MockTxManager = *txmmocks.TxManager[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
+
```

Comment:
there are multiple definitions of this in this PR - can we move them together and have them defined in one place?